repository/src/main/java/tech/ydb/yoj/repository/BaseDb.java

```diff
@@ -11,4 +11,8 @@ static <T> T current(Class<T> type) {
     }
 
     <T extends Entity<T>> Table<T> table(Class<T> c);
+
+    default <T extends Entity<T>> Table<T> table(Class<T> c, String name) {
```

Comment:
Please mark this method as `@ExperimentalApi(issue="https://github.com/ydb-platform/yoj-project/issues/32")`.